metadata_backend/api/handlers/object.py

```diff
@@ -104,6 +104,7 @@ async def post_object(self, req: Request) -> Response:
         _allowed_csv = {"sample"}
         _allowed_doi = {"study", "dataset"}
         schema_type = req.match_info["schema"]
+        LOG.info(f"Creating {schema_type} object")
```

Comment:
this looks more as  a `.debug` log to me, or is there any reason we need to make it `.info` ?